.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+### 📝 Issue Title: [Title of your issue]
+
+### 🔍 Scope
+
+[Clearly define the scope and objectives of the issue. What problem are you trying to solve or what feature are you looking to implement?]
+_Example: "Add a search bar to the dashboard to improve user navigation."_
+
+### 🔄 How to Implement the Issue
+
+[Provide detailed steps or approach for implementing the proposed solution]
+
+- Step 1:
+- Step 2:
+- Step 3:
+
+### ✅ Acceptance Criteria
+
+[List specific, measurable conditions that must be met to consider this issue complete]
+
+- [ ] Requirement 1
+- [ ] Requirement 2
+- [ ] Requirement 3
+
+### 📸 Additional Context
+
+[Add any additional information, screenshots, or relevant details that might help in understanding the issue]
+
+### Contribution Guidelines
+
+- Please ensure you read and follow the contribution guidelines in the project's [README](https://github.com/ByteBuildersLabs/ByteBeastsFrontend/blob/main/README.md) file
+
+### ✅ Checklist
```

Comment:
can you delete this please?

Reply:
just to confirm, only checklist section or everything from line 30 down ?

Reply:
everything from line 32 down

Reply:
fixed